packages/bpk-component-accordion/index.js

```diff
@@ -22,12 +22,10 @@ import BpkAccordion from './src/BpkAccordion';
 import BpkAccordionItem from './src/BpkAccordionItem';
 import withSingleItemAccordionState from './src/withSingleItemAccordionState';
 import withAccordionItemState from './src/withAccordionItemState';
-import themeAttributes from './src/themeAttributes';
 
 export {
   BpkAccordion,
   BpkAccordionItem,
   withSingleItemAccordionState,
   withAccordionItemState,
-  themeAttributes,
```

Comment:
same as above. is this needed/related?

Reply:
yep same as above, just cleaning up as we don't need this anymore